requirements.txt

```diff
@@ -3,5 +3,5 @@ hydra-genetics==0.15.0
 jinja2==3.0.1
 pandas>=1.3.1
 singularity==3.0.0
-snakemake>=7.8.3
+snakemake>=7.8.3,<8
```

Comment:
Assuming snakemake abides by semantic versioning, this should be fine.
![image](https://github.com/hydra-genetics/cnv_sv/assets/2573608/4b22807b-cd16-4eda-a9f1-c3cd4bbe9706)